### PR TITLE
Created open-source.mdx

### PR DESCRIPTION
**Pull request by lavanyayangala**
Added opensource doc contribution information.
Thankyou for making part of the first file collaboration 👩‍💻😊

## Fixes Issue

Closes #<issue_number>

### CODE REVIEW

File: docs/general-terminology/open-source.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Open Source
+edit: https://github.com/pbclife/gitopener.vercel.app/edit/main/docs/open-source.mdx
```
Comment (sboy99) [inline docs/general-terminology/open-source.mdx:3]: Remove this edit key and link the link will be auto generated, in the beginning I was doing it explicitly but in new updates I made it automatic

File: docs/general-terminology/open-source.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+# Open Source
```
Comment (sboy99) [inline docs/general-terminology/open-source.mdx:6]: I think What is Open Source? would be most prominent as its a question.

**Review by sboy99 — CHANGES_REQUESTED** [review]
Hey Thank you for making your time for this contribution, these changes are requested, just update them

File: docs/general-terminology/open-source.mdx (hunk 2 above)
Comment (lavanyayangala) [reply docs/general-terminology/open-source.mdx:6]: Yeah changing now..

Comment (sboy99) [reply docs/general-terminology/open-source.mdx:6]: I am really sorry actually I forgot that time the changes will work on guides directory. Will you move `general terminology` folder inside `docs/guides` directory.

Comment (lavanyayangala) [reply docs/general-terminology/open-source.mdx:6]: I made changes but it added to my forked repo may i know the steps to follow next

Comment (sboy99) [reply docs/general-terminology/open-source.mdx:6]: Just move your created folder and file inside `guides` --> `/docs/guides` instead of just `/docs`

Comment (lavanyayangala) [reply docs/general-terminology/open-source.mdx:6]: How to? Please guide this part...

Comment (sboy99) [reply docs/general-terminology/open-source.mdx:6]: - create a directory named `general-terminology` inside `docs/guides` directory
- inside `general-terminology` create a file `open-source.mdx`
- add a title in the front-matter section
- in the body add a heading followed by your contribution

Comment (lavanyayangala) [reply docs/general-terminology/open-source.mdx:6]: Done the changes but if i do the Pull request the code is storing in my fork repo is it correct?
How to add it to main one?

Comment (sboy99) [reply docs/general-terminology/open-source.mdx:6]: Yeah 👍🏾

Comment (lavanyayangala) [reply docs/general-terminology/open-source.mdx:6]: Done please check

**Pull request closed without merging**